actions/plugins.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Client4} from 'mattermost-redux/client';
+
+export async function doPluginAction(pluginId, requestURL, body) {
+    await Client4.executePluginIntegration(pluginId, requestURL, body);
```

Comment:
While calling the plugin directly is fine for plugin, that won't work for Apps and legacy integrations. At least for the former we will need the server to proxy the request. 


But that can be addressed outside of this PR.